charlesinto/iReporter
# #162527934 get all red-flags

#### What does this PR do?

- get all red-flags record

#### Description of Task to be completed?
- implement jwt authentication
- verify user token
- get all red-flags for an admin user
- get all red-flags for a specific user

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `GET` request to `http://localhost:7000/api/v1/red-flags/`
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint response should be structured like this- 



`

#### What are the relevant pivotal tracker stories?
[#162527934](https://www.pivotaltracker.com/story/show/162527934)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/26615085/49698305-3cf95300-fbc2-11e8-98c4-44d9124bf0a5.png)

#### Questions:
N/A
